examples/basic-env-vars.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: chaostoolkit-env
+  namespace: chaostoolkit-run
+data:
+  FIRST: "Jane"
+  SECOND: "Doe"
```

Comment:
I would suggest using more meaningful variables, like firstname and lastname, wdyt ?

Reply:
fixed